ui/src/templates/utils/index.ts

```diff
@@ -73,3 +73,20 @@ export const getLabelRelationships = (resource: {
 
 export const getIncludedLabels = (included: {type: TemplateType}[]) =>
   included.filter((i): i is LabelIncluded => i.type === TemplateType.Label)
+
+// See https://github.com/influxdata/community-templates/
+// an example of a url that works with this function:
+// https://github.com/influxdata/community-templates/tree/master/csgo
+export const getTemplateNameFromGithubUrl = (url: string): string => {
+  if (!url.includes('https://github.com/influxdata/community-templates/')) {
```

Comment:
why is this? templates can come from anywhere

Reply:
for the first version, i only want to target this repo: https://github.com/influxdata/community-templates/ and i'm paranoid of accepting something that we can't parse

Reply:
so the pkgs the UI will never parse, at least not in the next long while. The chellange with poeple going to influx, is they need to know to provide the `https://raw.githubusercontent.com/influxdata/community-templates/master/influxdb2_oss_metrics/influxdb2_oss_metrics.yml` instead of the plain `https://github.com` one. I raised this question in our dev inception, but I didn't hear any firm direction for if we will be translating github links to retrieve the raw contents or not, and if so, who does it 🤷.

Reply:
@jsteenb2 thanks for the explanation, that makes sense.

I think it makes most sense for the UI to handle this complexity. Ideally, we'd accept a whole bunch of different link formats and file formats, and as long as we can infer the name from what the user provides us, the UI should be fine translating that into the interface the backend expects. 

seems like the a good way to split responsibilities - part of the job of the UI is to make it easy for people to get data into the backend engine. if people want to do a lot of work repeatedly or quickly, they can use the more constrained cli api and write scripts to do that. if they just want to drag some random resource they got from influx into our UI to experiment or practice or just because they're less technical, that's also supported and doable.

Reply:
can you describe the specifics of what you are describing? I'm not following what you are proposing

Reply:
@jsteenb2 if a user gives us this url `https://github.com/influxdata/community-templates/tree/master/csgo` or this url https://github.com/influxdata/community-templates/blob/master/csgo/csgo.yml, or basically any link inside this directory: https://github.com/influxdata/community-templates/tree/master/csgo, we should be able to figure out they're trying to give us the yaml file.

Reply:
@hoorayimhelping, this is going to require significant investment of engineering effort to accomplish. The example you gave, may or may not hold weight. Not all templates are created this way. Take a gander at the community templates. They aren't all the same. Some are json, some are yaml, some have different namign conventions. Its more open that what you eluded too. 

I'd say punt on this complexity altogether. It'd be easier to just inform the user of what specific url we are looking for.